composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nypl-checkout-request-service",
+  "name": "nypl/checkout-request-service",
```

Comment:
Will this affect any dependencies of the `checkout-request-service`?

Reply:
Sorry overlooked this question.. No. If some other codebase *included* checkout-request-service as a composer module, this might disrupt that. But we don't really do that. The edit above is being made because composer requires this pattern for package names now, so if there was a codebase impacted by this change, it would probably also stumble on the old invalid package name.